data/net/minecraft/commands/arguments/coordinates/BlockPosArgument.mapping

```diff
@@ -2,8 +2,12 @@ CLASS net/minecraft/commands/arguments/coordinates/BlockPosArgument
 	METHOD getLoadedBlockPos (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)Lnet/minecraft/core/BlockPos;
 		ARG 0 context
 		ARG 1 name
+	METHOD getOrLoadBlockPos (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)Lnet/minecraft/core/BlockPos;
+		ARG 0 context
+		ARG 1 argumentName
 	METHOD listSuggestions (Lcom/mojang/brigadier/context/CommandContext;Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 context
 		ARG 2 builder
+	METHOD parse (Lcom/mojang/brigadier/StringReader;)Ljava/lang/Object;
```

Comment:
Please revert this change and the others similar to it in the same package, as this adds nothing. (This will likely be removed by a `sanitizeData` pass.)